guides/source/active_job_basics.md

```diff
@@ -10,6 +10,7 @@ After reading this guide, you will know:
 
 * How to create jobs.
 * How to enqueue jobs.
+* How to pass arguments to a jobs.
```

Comment:
Oh I there is a typo there too, I missed it. 
Do you suggest I remove the whole line ?